cmd/inspectImages.go

```diff
@@ -0,0 +1,143 @@
+// Copyright © 2018 openSUSE opensuse-project@opensuse.org
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"errors"
+	"os"
+	"path"
+	"path/filepath"
+
+	"github.com/openSUSE/helm-mirror/formatter"
+	"github.com/openSUSE/helm-mirror/service"
+	"github.com/openSUSE/helm-mirror/utils"
+
+	"github.com/spf13/cobra"
+)
+
+var imagesFile string
+var output string
+var target string
+
+const imagesDesc = `Extract all the images of the Helm Chart or
```

Comment:
The same for all these `const`. That being said, maybe since they have such a big string, we can ignore this rule here.